kubernetes/types.go

```diff
@@ -5,9 +5,9 @@ import (
 	"strings"
 
 	"github.com/kiali/kiali/config"
-	"k8s.io/api/apps/v1beta1"
-	autoscalingV1 "k8s.io/api/autoscaling/v1"
-	v1 "k8s.io/api/core/v1"
+	apps_v1 "k8s.io/api/apps/v1"
+	autoscaling_v1 "k8s.io/api/autoscaling/v1"
```

Comment:
MY EYES! MY EYES!! OK, this is how I would want it to be, but following your pattern, why didn't you call this autos_v1? You are arbitrarily abbreviating some places but not others, it seems.

Reply:
`auto_v1` *

Reply:
OK, so because "auto" is not very descriptive, and there are not many occurrences, I am keeping the explicit naming.